libfdt/fdt_ro.c

```diff
@@ -466,6 +466,17 @@ const void *fdt_getprop_namelen(const void *fdt, int nodeoffset,
 	return prop->data;
```

Comment:
Hmm.. I think I need a bit more convincing that this is sufficiently widely useful to be worth adding.  It is, after all, just a wrapper over two consecutive calls.